pypsa/optimization/constraints.py

```diff
@@ -415,40 +859,88 @@ def p_previous(idx):
             active.index, columns=fix_i
         )
         m.add_constraints(
-            lhs, ">=", rhs, name=f"{c}-fix-{attr}-ramp_limit_down", mask=mask
+            lhs, ">=", rhs, name=f"{c}-fix-non-comm-{attr}-ramp_limit_down", mask=mask
+        )
+
+    # ----------------------- Extendable but non committable Generators ----------------------------- #
```

Comment:
Why is this section needed?
For non-commitable generators I didn't expect much changes no? the previous constrains should be enough